pkg/volume/csi/csi_attacher_test.go

```diff
@@ -360,16 +361,20 @@ func TestAttacherWithCSIDriver(t *testing.T) {
 				t.Log("plugin is not attachable")
 				return
 			}
+			var wg sync.WaitGroup
 
 			go func(volSpec *volume.Spec, expectAttach bool) {
 				attachID, err := csiAttacher.Attach(volSpec, types.NodeName("node"))
+				defer wg.Done()
```

Comment:
This can actually be executed before the call to `Add()`, causing a panic

Reply:
good catch. fixed